README.md

```diff
@@ -75,6 +75,7 @@
 | [prometheus-mysqld-exporter](./images/prometheus-mysqld-exporter) | `cgr.dev/chainguard/prometheus-mysqld-exporter` | experimental | [![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/prometheus-mysqld-exporter.build.status.latest.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/prometheus-mysqld-exporter:latest) |
 | [prometheus-node-exporter](./images/prometheus-node-exporter) | `cgr.dev/chainguard/prometheus-node-exporter` | stable | [![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/prometheus-node-exporter.build.status.latest.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/prometheus-node-exporter:latest) |
 | [prometheus-postgres-exporter](./images/prometheus-postgres-exporter) | `cgr.dev/chainguard/prometheus-postgres-exporter` | experimental | [![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/prometheus-postgres-exporter.build.status.latest.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/prometheus-postgres-exporter:latest) |
+| [pulumi](./images/pulumi) | `cgr.dev/chainguard/pulumi` | experimental | [![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/pulumi.build.status.latest.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/pulumi:latest) |
```

Comment:
does it need to be experimental? do we know of issues with the image or do expect breaking changes?

Reply:
let me make sure it works in all the supported languages :)

Reply:
switched to stable